app/controllers/idv/enter_password_controller.rb

```diff
@@ -178,13 +178,20 @@ def next_step
       end
     end
 
+    def scrub_message(message)
+      filtered_message = message.dup
+      filtered_message.gsub!(/sponsorID \d+/i, 'sponsorID [FILTERED]')
+      filtered_message
+    end
+
+
     def handle_request_enroll_exception(err)
       analytics.idv_in_person_usps_request_enroll_exception(
         context: context,
         enrollment_id: err.enrollment_id,
         exception_class: err.class.to_s,
         original_exception_class: err.exception_class,
-        exception_message: err.message,
+        exception_message: scrub_message(err.message),
```

Comment:
Shouldn't this PR be scrubbing the logs in more places? I thought we would have to scrub it from the logs in places where proofer methods are used related to USPS endpoints getIppFacilityList // optInIPPApplicant // updateIPPApplicant // requestEnrollmentCode